backend/src/tasks/ecs-client.ts

```diff
@@ -1,5 +1,6 @@
 import { ECS } from 'aws-sdk';
 import { SCAN_SCHEMA } from '../api/scans';
+import * as Docker from 'dockerode';
```

Comment:
I switched to using imports from the previous conditional `require` because then we get typing with the dockerode library. I think that convenience is worth the small increase in bundle size with bundling dockerode (140 KB)